src/test/resources/simulators/avdManager.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as Q from "q";
+import {DeviceHelper} from "./deviceHelper";
+
+export interface IAVDManager {
+    // Intends to simulate: android create avd -n <name> -t <targetID> [-<option> <value>] ...
+    create(avdId: string): Q.Promise<void>;
+    // Intends to simulate: emulator -avd WVGA800 -scale 96dpi -dpi-device 160
+    launch(avdId: string): Q.Promise<string>;
+    createAndLaunch(avdId: string): Q.Promise<string>;
+    createAndLaunchAll(...avdIds: string[]): Q.Promise<string[]>;
+}
+
+interface IDeviceState {
```

Comment:
I don't think we should leave stuff half done for future work. Either implement it, or remove it.